jmxtrans-output/jmxtrans-output-influxdb/src/test/java/com/googlecode/jmxtrans/model/output/InfluxDbWriterTests.java

```diff
@@ -118,13 +121,42 @@ public void pointsAreWrittenToInfluxDb() throws Exception {
 	}
 
 	@Test
+	public void emptyCustomTagsDoesntBotherWrite() throws Exception {
+		InfluxDbWriter writer = getTestInfluxDbWriterWithDefaultSettings();
+		writer.doWrite(dummyServer(), dummyQuery(), results);
+		verify(influxDB).write(messageCaptor.capture());
+		BatchPoints batchPoints = messageCaptor.getValue();
+
+		assertThat(batchPoints.getDatabase()).isEqualTo(DATABASE_NAME);
+		List<Point> points = batchPoints.getPoints();
+		assertThat(points).hasSize(1);
+	}
+
+	@Test
+	public void customTagsAreWrittenToDb() throws Exception {
+		ImmutableMap<String, String> tags = ImmutableMap.<String, String>builder().put("customTag", "customValue").build();
```

Comment:
More compact way to initialize this map: `ImmutableMap.of("customTag", "customValue")` (I think... I did not actually check it).